### DISCUSSION TITLE
How to backup properly?

### DISCUSSION THREAD

**Discussion by whisperdancer** [simple-login/app#1368] [Self-hosting question] [answered]
What is your recommended best practice for backing up the data for an SL instance. Would a backup of the sl folder and its child folders be sufficient? Do I also need to do a db dump?

Thank you!

**Comment by nguyenkims** [comment, marked as answer]
@whisperdancer the most important data is the database and you can either backup the database folder (at `/sl/db`) or use db dump.

**Comment by whisperdancer** [comment]
Thank you!
